cg/meta/invoice.py

```diff
@@ -12,27 +12,29 @@ def __init__(self, db: Store, lims_api: lims.LimsAPI, invoice_obj: models.Invoic
         self.db = db
         self.lims_api = lims_api
         self.log = []
-        self.invoice_obj = invoice_obj #= self.invoice(invoice_id)
+        self.invoice_obj = invoice_obj
         self.customer_obj  = invoice_obj.customer
 
     def prepare_contact_info(self, costcenter):
         msg = f'Could not open/generate invoice. Contact information missing in database for customer {self.customer_obj.internal_id}. See log files.'
         if costcenter.lower() == 'kth':
-            contact_customer = self.db.customer('cust999')
+            customer = self.db.customer('cust999')
         else:
-            contact_customer = self.customer_obj
-        contact_user = self.db.user(contact_customer.invoice_contact)
+            customer = self.customer_obj
+        user = customer.invoice_contact
 
-        if not (contact_customer and contact_user):
+        if not user:
             self.log.append(msg)
             return None
+
         contact = {
-                'name': contact_user.name,
-                'email': contact_user.email,
-                'customer_name': contact_customer.name,
-                'reference': contact_customer.invoice_reference,
-                'address': contact_customer.invoice_address,
+                'name': user.name if user else '',
```

Comment:
```suggestion
                'name': user.name,
```
Om vi saknar user så borde vi redan hoppat ur på rad 28?